ansible_collections/arista/cvp/molecule/cv_image_v3/test_cv_image_v3.yaml

```diff
@@ -0,0 +1,138 @@
+---
+- name: Test cv_image_v3
+  hosts: CloudVision
+  connection: local
+  gather_facts: no
+  vars:
+    # Make sure given image does not exist on CVP
+    image_name: "EOS-4.25.4M.swi"
+  tasks:
+
+  ### Run all the tasks sequentially ###
+
+    - name: "Create bundle {{inventory_hostname}}"
+      vars:
+        ansible_command_timeout: 1200
+        ansible_connect_timeout: 600
+      arista.cvp.cv_image_v3:
+        mode: bundle
+        action: add
+        bundle_name: Test_bundle
+        image_list:
+          - "{{image_name}}"
+      register: CREATE_BUNDLE_DATA
+    - assert:
+        that:
+          - CREATE_BUNDLE_DATA.changed == true
+          - CREATE_BUNDLE_DATA.data.data == "Bundle : Test_bundle successfully created"
+          - CREATE_BUNDLE_DATA.failed == false
+
+    - name: "Update bundle {{inventory_hostname}}"
+      vars:
+        ansible_command_timeout: 1200
+        ansible_connect_timeout: 600
+      arista.cvp.cv_image_v3:
+        mode: bundle
+        action: add
+        bundle_name: Test_bundle
+        image_list:
+          - "{{image_name}}"
+      register: UPDATE_BUNDLE_DATA
+    - assert:
+        that:
+          - UPDATE_BUNDLE_DATA.changed == true
+          - UPDATE_BUNDLE_DATA.data.data == "Image bundle updated successfully."
+          - UPDATE_BUNDLE_DATA.failed == false
+
+    - name: "Gather CVP bundle information facts {{inventory_hostname}}"
+      vars:
+        ansible_command_timeout: 1200
+        ansible_connect_timeout: 600
+      arista.cvp.cv_image_v3:
+        mode: bundle
+        action: get
+      register: BUNDLE_DATA
+    - assert:
+        that:
+          - BUNDLE_DATA.changed == false
+          - BUNDLE_DATA.data.bundles[0].name == "Test_bundle"
+          - BUNDLE_DATA.data.bundles[0].imageIds == ["{{image_name}}"]
+
+    - name: "Delete bundle {{inventory_hostname}}"
+      vars:
+        ansible_command_timeout: 1200
+        ansible_connect_timeout: 600
+      arista.cvp.cv_image_v3:
+        mode: bundle
+        action: remove
+        bundle_name: Test_bundle
+        image_list:
+          - "{{image_name}}"
+      register: DELETE_BUNDLE
+    - assert:
+        that:
+          - DELETE_BUNDLE.changed == true
+          - DELETE_BUNDLE.data.data == "success"
+          - DELETE_BUNDLE.failed == false
```

Comment:
Can we try to get the same bundle and make sure it is deleted?

Reply:
Fixed! 
Here we are checking the name of first element of the bundle list not equals to "Test_bundle" (Negative Test).